src/adminSite.ts

```diff
@@ -106,6 +106,9 @@ class DefaultAdminSite {
         if (isDecimalType(column.type)) {
           cleanedValues[property] = parseFloat(value)
         }
+        if (isEnumType(column.type) && value === '') {
```

Comment:
this `value === ''` is impossible, we already know that `value` is truthy here. Probably you wanted to do that in the `if (!value)`?